app/views/tag/show.html.erb

```diff
@@ -24,10 +24,10 @@
       <% end %>
       <div class="tag-buttons">
         <div class="btn-group" role="group">
-          <% if current_user && current_user.following(@title) %>
```

Comment:
I think it's good that we're maintaining consistency by using `params[:id]` instead of `@title`.
There is another reference to `@title` on line 8 in the `<h1>` tag. Should we change this to be `params[:id]` as well?

Reply:
No, we should not. I brought this up to @jywarren a few posts back and he mentioned that this `@title` is referencing the header title for the web page. Line 8 is copied/pasted below for reference;

`<h1 style="margin-top:6px;"><%= @title %></h1>`

Reply:
Sounds good, thanks for clarifying!

Reply:
You are very welcome. I can't take all the credit. @jywarren mentioned that to me, just happy to pass it along.